src/H5.c

```diff
@@ -288,6 +291,14 @@ H5_init_library(void)
     }
     /* clang-format on */
 
+    /* Check for HDF5_USE_SELECTION_IO env variable */
+    env_use_select_io = HDgetenv("HDF5_USE_SELECTION_IO");
```

Comment:
This should probably use a constant defined in H5public.h, like the other environment variable strings.

Reply:
This is meant to be a temporary solution that will be removed before 1.14.